openmetadata-ui/src/main/resources/ui/src/styles/variables.less

```diff
@@ -12,6 +12,8 @@
  */
 
 @primary-color: #7147e8;
+@primary-color-II: #8d6af1;
```

Comment:
We should not create variant of primary-colour
If you want you can create it for lineage-specific like: `lineage-primary-color`

Reply:
removed variable used this colour only where it is required